src/SDKs/Network/Network.Tests/Network.Tests.csproj

```diff
@@ -35,4 +35,12 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  
+  <ItemGroup>
```

Comment:
@EvgenyAgafonchikov you do not need this, it adds warning as your test project implicitly reference these, else no tests would have run on the first place.
Remove these redundant entries